gui/unit-syndromes.lua

```diff
@@ -422,14 +421,20 @@ function UnitSyndromes:init()
             frame={l=0, b=0},
             label='Back',
             auto_width=true,
-            key='KEYBOARD_CURSOR_LEFT',
+            key='LEAVESCREEN',
             on_activate=previous_page,
             active=is_not_main_page,
             enabled=is_not_main_page,
         },
     }
 end
 
+function UnitSyndromes:onInput(keys)
+    if keys._R_MOUSE_DOWN then
+        self:previous_page()
```

Comment:
Make sure you return true if you handled the r click, otherwise ZScreen will still close the window